fuse/data/utils/collates.py

```diff
@@ -134,7 +134,8 @@ def _batch_dispatch(
             # use special handler if specified
             batch_dict[key] = self._special_handlers_keys[key](collected_values)
         elif isinstance(
-            collected_values[0], (torch.Tensor, np.ndarray, float, int, str, bytes)
+            collected_values[0],
```

Comment:
Do you really need it?
It will collect each element separately 
If you do it - I suggest adding a flag, cause sometimes we want to avoid it.
Or maybe add a special handler whenever you want this behavior.

Reply:
I'll remove it from this PR